apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -70,13 +88,10 @@ class StudentAssessmentOverviewTable extends Component {
   };
 
   studentAnswerColumnFormatter = (studentAnswers, {rowData, rowIndex}) => {
-    const selectStudentAnswers = this.props.studentAnswerData[0].studentAnswers[rowIndex];
+    const studentAnswersForRow = this.props.studentAnswerData[0].studentAnswers[rowIndex];
```

Comment:
is `this.props.studentAnswerData[0]` this still needed? we discussed this possibly being available in `rowData` (or another param provided by reactabular)